Create footer

andrewkdinh opened the pull request:
Replicated this mockup footer

<img width="1332" alt="footer" src="https://user-images.githubusercontent.com/45807756/82299133-ab8c3400-99a4-11ea-8abb-ff34bc70ef80.png">

andrewkdinh committed:
Create footer

PotatoParser commented on partials/footer.hbs:
```diff
@@ -2,28 +2,57 @@
+          <li><a href="." rel="noopener">Home</a></li>
+          <li><a href="about" rel="noopener">About</a></li>
+          <li><a href="events" rel="noopener">Events</a></li>
+          <li><a href="apply" rel="noopener">Apply</a></li>
```
Switch href to absolute path: `{{ @site.url }}/<route to your link>`

PotatoParser commented on partials/footer.hbs:
```diff
@@ -2,28 +2,57 @@
+        <ul>
+          <li><a href="micromouse" rel="noopener">Micromouse</a></li>
+          <li><a href="hope" rel="noopener">HOPE</a></li>
+          <li><a href="industry" rel="noopener">Industrial Relations</a></li>
```
Switch href to absolute path: `{{ @site.url }}/<route to your link>`

PotatoParser commented on src/sass/components/_footer.scss:
```diff
@@ -1,67 +1,105 @@
-    &:nth-child(2) {
-      display: none;
+.footer-mailing-list button {
+  background-color: transparent;
```
Add `border-style: solid;` instead of using the default of `outset`

PotatoParser commented on partials/footer.hbs:
```diff
@@ -2,28 +2,57 @@
-    </p>
+  </div>
+  <div class="footer-copyright">
+    <p>{{@site.title}} &copy; {{date format="YYYY"}}</p>
```
I'm not sure if this is the correct copyright that we want to see. I think the site title is different from our actual copyright. `© Copyright 2020 IEEE Student Branch at UC Berkeley`

PotatoParser commented on src/sass/components/_footer.scss:
```diff
@@ -1,67 +1,105 @@
+  display: block;
+  font-size: 18px;
+  font-weight: bolder;
+  color: rgba(253, 180, 21);
```
This color is a little warm, the hexcode for the color in the image is `#fed966`

PotatoParser commented on src/sass/components/_footer.scss:
```diff
@@ -1,67 +1,105 @@
+  margin: 10px 0px;
+}
+
+.footer-col a {
```
Needed hover animation (to help users see what they are hovering over)

PotatoParser commented on src/sass/components/_footer.scss:
```diff
@@ -1,67 +1,105 @@
 
-    &:nth-child(2) {
-      display: none;
+.footer-mailing-list button {
```
There seems to be a small border-radius on the image, perhaps adding `border-radius: 2px;` would work.

PotatoParser reviewed (changes requested):
Overall a very clean job! There are just a few visual elements that are off.

andrewkdinh committed:
Footer fixes

- Correct copyright
- Change relative links to absolute
- Add hover animations for links

encadyma reviewed (approved):
Any reason why the mailing list border is outsetted? Otherwise, it looks good.

sulattphone reviewed (approved):
Looks solid.
I wonder how it would look if we have our IEEE logo right in the middle of the footer? Can we see that?

andrewkdinh committed:
Added IEEE logo

Added IEEE logo to middle of footer. Logo dissapears on mobile
